src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -548,28 +547,25 @@ private async Task FetchIconAsync()
                     // to support downloading on a worker thread, we need to download the image
                     // data and put into a memorystream. Then have the BitmapImage decode the
                     // image from the memorystream.
-#pragma warning disable CA2000 // Dispose objects before losing scope
-                    var memoryStream = new MemoryStream();
-#pragma warning restore CA2000 // Dispose objects before losing scope
-
-                    // Cannot call CopyToAsync as we'll get an InvalidOperationException due to CheckAccess() in next line.
-                    stream.CopyTo(memoryStream);
-                    iconBitmapImage.StreamSource = memoryStream;
-
-                    try
+                    using (var memoryStream = new MemoryStream())
```

Comment:
[NuGet.PackageManagement.UI](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.UI/NuGet.PackageManagement.UI.csproj#L25) has `LangVersion` set to 8. Can we use simplify this [using](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations)?

Reply:
i didn't know it was a thing to do it like this:
```
    using var r1 = new AResource();
    using var r2 = new AResource();
    r1.UseIt();
    r2.UseIt();
```

seems like that could eliminate a number of nested curly brackets...

Reply:
Yes, Rob that is my understanding. As per doc, `It tells the compiler that the variable being declared should be disposed at the end of the enclosing scope`. Nikolche suggested me to use C# 8 feature in https://github.com/NuGet/NuGet.Client/pull/3824#discussion_r551639338. This is optional Rob; the changes look good to me.